dist/render.js

```diff
@@ -0,0 +1,13 @@
+class Render{
+    constructor(data){
+        this.data = data
+    }
+    
+    renderData(){
+        $('#players').empty()
```

Comment:
The render method should receive an argument. If it's rendering the object data it will enforce us to create a new instance (or change the object data) each time we want to render data.